### DISCUSSION TITLE
如何安装 LSPosed 到 Chrome OS 上？

### DISCUSSION THREAD

**Discussion by 1457384613gh** [LSPosed/LSPosed#1960] [Q&A] [answered]
Chrome OS volteer on Brunch with Android 11。
如何安装 LSPosed 到 Chrome OS 上？

**Comment by yujincheng08** [comment, marked as answer]
我没有 Chrome OS 设备，无法测试。

**Reply by 1457384613gh** [reply]
https://github.com/sebanc/brunch

**Reply by yujincheng08** [reply]
> Virtual Machines are not supported

**Reply by 1457384613gh** [reply]
https://fydeos.com/download/vm

**Reply by yujincheng08** [reply]
https://community.fydeos.com/t/topic/808/6

**Reply by 1457384613gh** [reply]
你只用 hyper-V，不使用别的平台的虚拟机？
好吧，没关系，可以理解，因为我也是。
